contrib/bmp2/README.md

```diff
@@ -0,0 +1,38 @@
+BMP2
+====
+
+![BMP2](http://ebrombaugh.studionebula.com/embedded/bmp2/bmp2.jpg)
+
+#### BMP2 assembled board
+
+### What is it?
+
+This is a revised layout for the Black Magic Probe Mini from Blacksphere which provides connections for several styles of JTAG connectors. In addition to the 10-pin mini JTAG found on the original, it also has connections compatible with the original 20-pin 0.1" pitch JTAG connector as well as the ST-Link 6-pin 0.1" single row header.
```

Comment:
Please include a statement that this is not official native hardware, but a third party development. In hopes to decrease user confusion.